### DISCUSSION TITLE
new version release timeline?

### DISCUSSION THREAD

**Discussion by chy168** [hairyhenderson/gomplate#1395] [General]
I ran into the issue similar to #1333 but the fix seems not included in the latest version (3.10.0).
Any idea when the next stable version will be released?

thanks gomplate with that many cool features! ❤️

**Comment by hairyhenderson** [comment]
Hi @chy168, Sorry for the delay. I'm quite busy with my day-job and haven't had a lot of free time to spend on gomplate. I'm working on releasing v3.11.0 soon, but can't make any estimates or promises.

**Reply by chy168** [reply]
thank you for the time and effort! Just build binary from master and it works very well included the fix.
looking forward to the next release. 👍
